.github/workflows/ci_tests.yaml

```
@@ -134,9 +134,15 @@ jobs:
           pip install dist/*
 
       # Run the tests
-      - name: Test with pytest
+      - name: Run tests
+        if: github.event_name != 'schedule'
         run: make test PYTEST_EXTRA="-r P"
 
+      # Run full tests including doctests
+      - name: Run full tests
+        if: github.event_name == 'schedule'
```

Not sure if GitHub Actions uses `&` or `and`, but try something like this to run full tests on Wednesday only. Xref https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
```suggestion
        if: github.event_name == 'schedule' & github.event.schedule == '0 0 * * 3'
```

---

> Not sure if GitHub Actions uses `&` or `and`,

It's `&&` (https://docs.github.com/en/actions/learn-github-actions/expressions#operators).